sw.js

```diff
@@ -0,0 +1,140 @@
+// This is the service worker with the Cache-first network
+
+importScripts('cache_version.js');
```

Comment:
I am not sure if we should set a manual cache version. I think we should only fallback to the online version of the page in case of a service worker error (the current behavior). Setting a manual cache_version may stuck users in an old page version.